trace-protobuf/src/pb/stats.proto

```diff
@@ -0,0 +1,70 @@
+syntax = "proto3";
+
+import "gogo.proto";
+
+package pb;
+
+// protoc --gogofaster_out=. -I $GOPATH/src -I . stats.proto
+
+// StatsPayload is the payload used to send stats from the agent to the backend.
+message StatsPayload {
```

Comment:
One thing we've noticed in the past in [system-test](https://github.com/DataDog/system-tests) - is that these files are updated in the agent from time to time - would it make sense to have CI check that verifies if they are still in sync ?

Reply:
added a new CI workflow to diff our `span.proto`, `agent_payload.proto`, and `stats.proto` files against those present in datadog-agent. Fixed a couple missing fields that were added recently too.